warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/BulkIngestMapFileLoader.java

```diff
@@ -489,6 +498,40 @@ public void run() {
         log.info("Bulk map file loader shutting down.");
     }
     
+    // take ownership by renaming files and then running cleanup script
+    private void cleanJobDirectoriesOnStartup() throws IOException {
+        Path[] cleanupDirectories = getJobDirectories(new Path(workDir, jobDirPattern + '/' + CLEANUP_FILE_MARKER));
+        for (int i = 0; i < cleanupDirectories.length; i++) {
+            boolean success = false;
+            
+            try {
+                success = getFileSystem(srcHdfs).rename(new Path(cleanupDirectories[i], CLEANUP_FILE_MARKER),
+                                new Path(cleanupDirectories[i], REMOVE_FILE_MARKER));
+                log.info("Renamed " + cleanupDirectories[i] + '/' + CLEANUP_FILE_MARKER + " to " + REMOVE_FILE_MARKER);
```

Comment:
There's no need to "take ownership" here.  If multiple loaders try to move the files, hdfs provides the atomicity.  Adding another "state" for ownership is yet another state we need to check for at startup if we do it this way.